src/finn/core/modelwrapper.py

```diff
@@ -288,6 +288,46 @@ def find_consumer(self, tensor_name):
         except ValueError:
             return None
 
+    def find_consumers(self, tensor_name):
+        """Finds and returns a list of the nodes that consume tensor with
+        given name."""
+        consumers = []
+        for n in self._model_proto.graph.node:
+            for inp_tensor in n.input:
+                if inp_tensor == tensor_name:
+                    consumers.append(n)
+        if consumers != []:
+            return consumers
+        else:
+            return None
+
+    def find_successors(self, node):
```

Comment:
either rename to find_direct_successors (my recommendation) or implement actual find_successors -- in graph theory, a successor is *any* node reachable "downstream" from this node, so if you have a graph x -> y -> z -> t the successors of x are y, z, y. "direct successor" would be a better description of the current behavior.